.gitignore

```diff
@@ -24,7 +24,10 @@ DerivedData/
 .idea/
 Carthage
 .DS_Store
+.ruby-version
+.bundle/
 .build/
+vendor/
```

Comment:
I used this for `rbenv`, we probably should by default use it too.

Reply:
👍 
I've always used `rvm` out of habit, and I've thought about adding a gemset and keep forgetting about it

Reply:
Oh interesting. I only use `rbenv` because that's what we preferred on `fastlane`. Creatures of habit we are.